python/sglang/srt/model_executor/forward_batch_info.py

```diff
@@ -112,6 +112,14 @@ class ForwardBatch:
     token_to_kv_pool: BaseTokenToKVPool = None
     attn_backend: AttentionBackend = None
 
+    # Double Sparsity Settings
+    sorted_channels: List[torch.Tensor] = None
+    heavy_token_num: int = 256
+    sparse_decode_thresold: int = 4096
+    att_out_approx: torch.Tensor=None
+    mid_out: torch.Tensor=None
+    mid_o_logexpsum: torch.Tensor=None
```

Comment:
move all of these to `python/sglang/srt/layers/attention/double_sparsity_backend.py`